lib/src/sv_internal.h

```diff
@@ -420,6 +426,11 @@ bu_type_to_str(const bu_info_t *bu);
 char
 bu_type_to_char(const bu_info_t *bu);
 
+/**
+ * Converts a MediaSigningReturnCode to a SignedVideoReturnCode. */
+SignedVideoReturnCode
+media_signing_return_code_to_signed_video_return_code(int code);
```

Comment:
Since this is an internal function this name is a little bit too long. We use `svrc` as a short name for `SignedVideoReturnCode` internally. How about naming this `msrc_to_svrc(MediaSigningReturnCode msrc)`?

Note that you should for clarity use the full name instead of `int`